src/pull-request-metrics-table/PullRequestMetricsTable.tsx

```diff
@@ -3,11 +3,23 @@ import { Link as ReactDomLink, useParams } from 'react-router-dom';
 import Tooltip from '@mui/material/Tooltip';
 import Link from '@mui/material/Link';
 import format from 'date-fns/format';
-import { DataGrid, GridColDef, GridRenderCellParams } from '@mui/x-data-grid';
+import IsoIcon from '@mui/icons-material/Iso';
```

Comment:
To get super nitty... I tend to find these easier to read like this:

`import { IsoIcon, InsertDriveFileOutlinedIcon}  from '@mui/icons-material/Iso';`

This is my new tabs vs spaces.